pylds/tools.py

```diff
@@ -74,12 +74,23 @@ def draw_lagrangian_descriptor(LD, LD_type, grid_parameters, tau, p_value, norm
         string_title = ''
         print('Incorrect "LD_type". Valid options: forward, backward, total. Plot will appear without title')
     
-    ax.set_title(string_title, fontsize=12)
-    ax.set_xlabel('$x$', fontsize=18)
-    ax.set_ylabel('$y$', fontsize=18)
+    fig.suptitle(string_title, fontsize=14, y=1.04)
+    ax0.set_title('LD values')
+    ax0.set_xlabel('$x$')
+    ax0.set_ylabel('$y$')
+    
+    fig.colorbar(con0, ax=ax0, ticks=np.linspace(np.nanmin(LD),np.nanmax(LD),11),format='%.2f')
```

Comment:
Avoid function calls with long arguments for enhanced readability, please.
For instace, 
```
ticks_values = np.linspace(np.nanmin(LD), np.nanmax(LD), 11)
labels_format = '%.2f'
fig.colorbar(con0, ax=ax0, ticks=ticks_values,format=labels_format)
```

Reply:
Agreed for the ticks, with format I have the same resevation as above - not good for readability